src/Confluent.SchemaRegistry/SchemaId.cs

```diff
@@ -0,0 +1,298 @@
+// Copyright 2025 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using Confluent.Kafka;
+
+namespace Confluent.SchemaRegistry;
+
+/// <summary>
+///     Represents a schema ID or GUID.
+/// </summary>
+public class SchemaId
```

Comment:
Since a SchemaId object is created for every deserialized message, it would be ideal to make it a struct to reduce the number of heap allocations.

Reply:
Fixed